code/datums/martial/krav_maga.dm

```diff
@@ -130,12 +132,14 @@
 	if(check_streak(A,D))
 		return 1
 	log_combat(A, D, "punched")
+	var/obj/item/bodypart/affecting = D.get_bodypart(ran_zone(A.zone_selected))
+	var/armor_block = D.run_armor_check(affecting, "melee")
 	var/picked_hit_type = pick("punch", "kick")
-	var/bonus_damage = 10
+	var/bonus_damage = 0
 	if(!(D.mobility_flags & MOBILITY_STAND))
 		bonus_damage += 5
 		picked_hit_type = "stomp"
-	D.apply_damage(bonus_damage, A.dna.species.attack_type)
+	D.apply_damage(rand(5,10) + bonus_damage, A.dna.species.attack_type, affecting, armor_block)
```

Comment:
You do know that this doesn't factor in a species' base punch damage, right? You might wanna incorporate species.punchdamagelow in here somewhere.

Reply:
Implying that golems are gonna know Krav Maga.

Reply:
And also this is already partially how the damage works on krav maga already. The other main martial arts also do not use species punch damage. The only thing it carries is damage type for the sake of ethereals and I'm pretty sure thats it.

The way it worked before was that krav maga would do 10 damage on normal punch and 15 on kick because the bonus damage value was the only thing used in apply_damage. All I added was randomized values and retained the shape of the code.

I'm not really interested in breaking the mold here. Human punchdamagelow is like 1 which is dreadful anyway.